### ISSUE TITLE
Zindex changes are not applied

### ISSUE THREAD

**Issue by vlada-symphony** [eregnier/vue2-gmap-custom-marker#25]
When marker is displayed and z-index prop gets changed, the marker doesn't respect the change. I think the solution is to add watch (similar to the one for watching position changes) and call repaint when z-index prop changes.

**Comment by eregnier** [comment]
It should work. You may take a look at the live demo https://eregnier.github.io/vue2-gmap-custom-marker/dist/ where there is a sample of z-index live edit component that dynamically toggle on user inputs.

Tell me if it is ok or not.

Regards

**Comment by vlada-symphony** [comment]
Hey, thanks for answering. Demo works as expected, but I think this happens only when custom marker is defined with v-for. I updated demo code with something like this:

```
<gmap-custom-marker
  v-for="marker in myMarkers"
  :key="marker.key"
  :z-index="marker.zIndex" :marker="marker.location"
  :style="{ backgroundColor: marker.background }"
>
  <div class="zindex">
    <center>
      <p>
        <b>{{marker.zIndex}}</b>
      </p>
      <button @click="marker.zIndex++">+</button>
      <button @click="marker.zIndex--">-</button>
    </center>
  </div>
</gmap-custom-marker>
```

and `myMarkers` is defined like this:

```
data() {
  return {
    myMarkers: [
      {
        location: { lat: 50.4272265, lng: 2.95 },
        key: 0,
        background: "#bbdddd",
        zIndex: 50,
      },
      {
        location: { lat: 50.4272265, lng: 2.80 },
        key: 1,
        background: "#ddbbdd",
        zIndex: 51,
      },
    ],
  },
}
```

And then the bug reproduced. `<b>{{marker.zIndex}}</b>` updates its text correctly, but `z-index` isn't updated until the map is moved or zoomed in/out (which is why I assumed `repaint` method isn't called or something).

**Comment by eregnier** [comment]
solved in 5.5.4 version, npm is up to date with the fix.

**Comment by eregnier** [comment]
Thank you @vlada-symphony for help in this plugin improvement.
